RELEASING.md

```diff
@@ -317,6 +317,8 @@ list of such packages as of today:
 * `libs/cardano-ledger-test`
 * `libs/plutus-preprocessor`
 * `libs/ledger-state`
+* `libs/cardano-ledger-conformance`
```

Comment:
> Should we update the version in this package to 9.9.9.9, for consistency with the rest of the unreleased packages ?

Yes, we should. However, it is not strictly required, since the reason why 9.9.9.9 is there is to discourage people unfamiliar with a package from releasing such package that is not suppose to be released. In case of `cardano-ledger-conformance` it is impossible to release it since it depends on an SRP of MAlonzo code. I'll update it for consistency.

> Also it occurred to me that libs/constrained-generators is not mentioned in this list but also it's not part of the command listing the packages we're releasing. 

The plan for it is to be moved into another repo fairly soon, so I am not gonna worry about it.